src/components/SingleArticle.jsx

```diff
@@ -29,22 +29,25 @@ export const SingleArticle = () => {
 	return isError ? (
 		<p className="error-display">There has been an error!</p>
 	) : isLoading ? (
-		<p className="loading-display">Loading Articles!</p>
+		<p className="loading-display">Loading Article!</p>
 	) : (
 		<>
 			<section className="single-article">
 				<div className="single-article-headings">
 					<h2>{article.title}</h2>
 					<p className="single-article-author">{article.author}</p>
 				</div>
-				<div>
+				<div className="article-main">
 					<img src={article.article_img_url} alt={article.title} />
-				</div>
-				<div>
 					<p className="single-article-body">{article.body}</p>
 				</div>
 			</section>
-			<section></section>
+			<section className="article-comments">
```

Comment:
The way you've incorporated the comments into the page means your user has to wait for the article and all the comments to finish loading before they can view it. Can you set it up so that the article loads and renders what it has loading while the comments continue loading? If there are loads of comments, the user might be waiting for a while. Perhaps the _list of comments_ could be their own component?